data/path/path.go

```diff
@@ -83,6 +66,35 @@ func GetValue(value interface{}, path string) (interface{}, error) {
 	return GetValue(newVal, newPath)
 }
 
+func getFieldValueByName(object interface{}, name string) (interface{}, error) {
+	val := reflect.ValueOf(object)
+	if val.Kind() == reflect.Ptr {
+		val = val.Elem()
+	}
+
```

Comment:
Should we validate that this a struct like before?

Reply:
@fm-tibco  Done